README.md

```diff
@@ -63,6 +82,18 @@ At this point, a test application will be running on port `3001`.
 Be sure that you enter a valid Solid issuer before logging in
 , for example: `https://identity.demo-ess.inrupt.com`.
 
-## Environment-specific documentation
+# Issues & Help
+
+## Solid Community Forum
+
+If you have questions about working with Solid or just want to share what you’re working on, visit the [Solid forum](https://forum.solidproject.org/). The Solid forum is a good place to meet the rest of the community.
+
+## Bugs and Feature Requests
+
+- For public feedback, bug reports, and feature requests please file an issue via [Github](https://github.com/inrupt/solid-client-authn/issues/).
+- For non-public feedback or support inquiries please use the [Inrupt Service Desk](https://inrupt.atlassian.net/servicedesk).
+
+## Documentation
 
-- [For the browser](./docs/browser.md)
+* [Using the libraries from within the browser](./docs/browser.md)
+* [Inrupt documentation Homepage](https://docs.inrupt.com/)
```

Comment:
I would put this above the support section.

Reply:
I hear you, no problem with that, but today It's at the end on all the README files. If you want to go ahead and change all of them to keep them consistent be my guest, otherwise I think at the end is fine